automation/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:15.04
+
+# Install the following utilities (required by poky)
+RUN apt-get update && apt-get install -y build-essential chrpath curl diffstat gcc-multilib gawk git-core libsdl1.2-dev texinfo unzip wget xterm cpio file && rm -rf /var/lib/apt/lists/*
+
```

Comment:
It needs to be on every layer, because each `RUN` creates a docker layer and then the later layers are just overlaid on top, meaning that the moment a file is committed in one layer then you will always have to download it even if it's later deleted